airbyte-integrations/connectors/source-gitlab/source_gitlab/streams.py

```diff
@@ -403,7 +403,7 @@ class Users(GitlabChildStream):
 
 
 class Epics(GitlabChildStream):
-    primary_key = "iid"
+    primary_key = "id"
```

Comment:
Is it a breaking change ?

Reply:
I tested it with both old and new pk, full_refresh overwrite/append, all works fine. please let me know if I tested in a wrong way. Also this stream only full refresh, and ui settings show pk as empty in this case.